.github/workflows/ci-osx.yml

```diff
@@ -55,7 +55,7 @@ defaults:
 jobs:
   # 64-bit OSX build with clang and run tests:
   osx-x86-64:
-    runs-on: macos-13
+    runs-on: macos-15-intel
```

Comment:
Hmm, I didn't think they offered any Intel VMs for free; didn't see this one. I think we want to transition to arm at some point anyway but probably this was a little less work for now.